tests/test_template.py

```diff
@@ -80,4 +81,11 @@ def test_render_json_template_invalid_json_missing_value(monkeypatch):
     template_file = StringIO('{"type_of_muffin": {{ MUFFIN_TYPE }}}')
 
     with pytest.raises(ValueError):
+        render_json_template(template_file, **{"MUFFIN_TYPE": ""})
+
+
+def test_render_json_template_missing_value_raises(monkeypatch):
```

Comment:
Yep, tested implicitly by the test at L64 which tests that the happy path works.

Happy to split that out into two tests if being more explicit would be better. One test to check it renders correctly and another that it doesn't raise?